src/util/StreamerMode.cpp

```diff
@@ -1,28 +1,29 @@
-#include "StreamerMode.hpp"
+#include "util/StreamerMode.hpp"
 
-#include "Application.hpp"
 #include "common/QLogging.hpp"
-#include "messages/MessageBuilder.hpp"
-#include "providers/twitch/TwitchIrcServer.hpp"
 #include "singletons/Settings.hpp"
 #include "singletons/WindowManager.hpp"
 #include "widgets/helper/NotebookTab.hpp"
-#include "widgets/Notebook.hpp"
-#include "widgets/splits/Split.hpp"
-#include "widgets/Window.hpp"
 
 #ifdef USEWINSDK
 // clang-format off
+// These imports cannot be ordered alphabetically.
 #    include <Windows.h>
-
 #    include <VersionHelpers.h>
 #    include <WtsApi32.h>
-#    pragma comment(lib, "Wtsapi32.lib")
 // clang-format on
+#    pragma comment(lib, "Wtsapi32.lib")
+#else
+#    include "Application.hpp"
+#    include "messages/MessageBuilder.hpp"
+#    include "providers/twitch/TwitchIrcServer.hpp"
+#    include "widgets/Notebook.hpp"
+#    include "widgets/splits/Split.hpp"
+#    include "widgets/Window.hpp"
+
+#    include <QProcess>
```

Comment:
I would rather not use an else-scope for includes like this unless strictly necessary as I believe it reduces maintainability/readability.

Essentially how I evaluate it is: If the include is only required on a specific platform but doesn't break the build of other platforms, just include them always in the main build.